Source/WebKit/UIProcess/Cocoa/VideoFullscreenManagerProxy.h

```diff
@@ -43,7 +43,10 @@
 #include <wtf/WeakHashSet.h>
 #include <wtf/text/WTFString.h>
 
-#if PLATFORM(IOS_FAMILY)
+#if PLATFORM(WATCHOS)
+#include <WebCore/NullVideoFullscreenInterface.h>
+typedef WebCore::NullVideoFullscreenInterface PlatformVideoFullscreenInterface;
```

Comment:
I know you're just copying the pattern used on other platforms, but would a forward declaration of `NullVideoFullscreenInterface` be sufficient here to define the typedef?